djangocms_alias/admin.py

```diff
@@ -85,9 +95,18 @@ def delete_model(self, request, obj):
 
 
 @admin.register(AliasContent)
-class AliasContentAdmin(admin.ModelAdmin):
+class AliasContentAdmin(*alias_content_admin_classes):
     form = AliasContentForm
-    list_filter = (LanguageFilter,)
+    list_filter = (LanguageFilter, )
+    list_display = alias_content_admin_list_display
+    change_form_template = "admin/djangocms_alias/aliascontent/change_form.html"
+
+    # Add Alias category in the admin manager list and order field
+    def get_category(self, obj):
+        return obj.alias.category
+
+    get_category.short_description = 'category'
```

Comment:
This should be translatable as it is shown to the user

```suggestion
    get_category.short_description = _("category")
```

Reply:
Will need this adding to the top, the editor won't let me add it as a suggestion: 

`from django.utils.translation import gettext_lazy as _`